vendor/vendor.json

```diff
@@ -538,22 +556,40 @@
 			"versionExact": "v0.10.0"
 		},
 		{
-			"checksumSHA1": "2fkVZIzvxIGBLhSiVnkTgGiqpQ4=",
+			"checksumSHA1": "mZQPLm1vepJpOQzbGbT/FXolAXI=",
 			"path": "github.com/hashicorp/vault/api",
-			"revision": "9a60bf2a50e4dd1ba4b929a3ccf8072435cbdd0a",
-			"revisionTime": "2016-10-29T21:01:49Z"
+			"revision": "f7150ae74c42e860de5c882577768b7472b30565",
```

Comment:
This doesn't appear to be a correct vendor for Vault API v0.10.4. This revision is in the v0.11.0 and v0.11.0-beta1 tags but not v0.10.4. [1] What `govendor` command did you use?

1. https://github.com/hashicorp/vault/commit/f7150ae74c42e860de5c882577768b7472b30565